IOSum/src/serg/homework/de/PersonReader.java

```diff
@@ -0,0 +1,32 @@
+package serg.homework.de;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Comparator;
+
+public class PersonReader {
+    public static void main(String [] args) throws IOException {
+        try (BufferedReader br = new BufferedReader(new FileReader("people.txt"));
```

Comment:
Again, it is just a code, which read file people.txt and stores the result in "sordted_people.txt". What if I need to run this code for another files?